src/graphql/data/fetchGraphQL.ts

```diff
@@ -1,28 +0,0 @@
-import { Variables } from 'react-relay'
-import { GraphQLResponse, RequestParameters } from 'relay-runtime'
-
-const GRAPHQL_URL = process.env.REACT_APP_AWS_API_ENDPOINT
-
-if (!GRAPHQL_URL) {
-  throw new Error('AWS URL MISSING FROM ENVIRONMENT')
-}
-
-const headers = {
-  'Content-Type': 'application/json',
-}
-
-const fetchQuery = (params: RequestParameters, variables: Variables): Promise<GraphQLResponse> => {
-  const body = JSON.stringify({
-    query: params.text, // GraphQL text from input
-    variables,
-  })
-
-  return fetch(GRAPHQL_URL, { method: 'POST', body, headers })
-    .then((res) => res.json())
-    .catch((e) => {
-      console.error(e)
-      return { data: [] }
```

Comment:
Are you sure this "catch and return `{data:[]}`" behavior is done by the new network class?

Reply:
I don't think it is, looking into how to recreate